Code/MinimalLib/minilib.cpp

```diff
@@ -145,7 +145,9 @@ std::string JSMol::get_pickle() const {
     return "";
   }
   std::string pickle;
-  MolPickler::pickleMol(*d_mol, pickle);
+  MolPickler::pickleMol(*d_mol, pickle,
+                        PicklerOps::MolProps | PicklerOps::AtomProps |
+                            PicklerOps::BondProps | PicklerOps::PrivateProps);
```

Comment:
Since I think you mean for this to be all properties except the computed ones, this would be shorter and more future proof as:
```
  MolPickler::pickleMol(*d_mol, pickle, PicklerOps::AllProps ^ PicklerOps::ComputedProps);
```

Reply:
Done.